include/bluetooth/bluetooth.h

```diff
@@ -130,6 +130,11 @@ typedef void (*bt_ready_cb_t)(int err);
  * Enable Bluetooth. Must be the called before any calls that
  * require communication with the local Bluetooth hardware.
  *
+ * When CONFIG_BT_SETTINGS has been enabled and the application is not managing
+ * identities of the stack itself then the application must call settings_load()
+ * before the stack is fully enabled.
+ * See bt_id_create() for more information.
```

Comment:
```suggestion

* When @option{CONFIG_BT_SETTINGS} has been enabled and the application is not managing 
* identities of the stack itself then the application must call @ref settings_load() 
* before the stack is fully enabled. 
* See @ref bt_id_create() for more information.

```

Reply:
Thanks.

Just FYI: It is possible to do multiline suggestion, you just have to hold down the mouse-key and drag down across the lines that your comment should apply for.